backend/copr_backend/pulp.py

```diff
@@ -74,7 +82,12 @@ def request_params(self):
         """
         Default parameters for our requests
         """
-        return {"auth": self.auth, "timeout": self.timeout}
+        params = {"timeout": self.timeout}
+        if all(self.cert):
+            params["cert"] = self.cert
+        else:
+            params["auth"] = self.auth
```

Comment:
Just curious, can we add a check that only `copr` has read access to the key file?
If not, we can make sure at deployment time.